readme.md

```diff
@@ -0,0 +1,30 @@
+![Polylang](assets/polylang-logo.png)
+
+Welcome to the Polylang repository on GitHub. Here you can browse the source, look at open
+issues and keep track of development.
+
+If you are not a developer, please use the [Polylang plugin page](https://wordpress.org/plugins/polylang/) on [WordPress.org](https://wordpress.org).
+
+## [Pre-requisites](#pre-requisites)
+
+Before starting, make sure you have the following software installed and working on your machine:
+
+1. A local [WordPress](https://wordpress.org/support/article/how-to-install-wordpress/) (5.4 or later) instance
+2. [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) to clone the Polylang repository (or your fork of the Polylang repository).
+3. [NVM](https://github.com/nvm-sh/nvm) or [chocolatey](https://chocolatey.org/install) (on Windows OS) to install [Node.js](https://nodejs.org/en/download/) and [NPM](https://docs.npmjs.com/). They are required by [Webpack](https://webpack.js.org/guides/getting-started/) that Polylang uses to build and minifies CSS and javascript files.
```

Comment:
I don't use NVM or Chocolatey, so for me they are not required.  Node.js and NPM should be sufficient. Am I wrong?
Typo: **minify**